guides/common/modules/proc_registering-a-host-without-puppet.adoc

```diff
@@ -5,7 +5,7 @@ By default, the bootstrap script configures the host for content management and
 If you have an existing configuration management system and do not want to install Puppet on the host, use `--skip-puppet`.
 
 .Procedure
-* On {EL} 8, enter the following command:
+* On {EL} 9 or 8, enter the following command:
```

Comment:
```suggestion
* Enter the following command according to your operating system:
** {EL} 8 or 9:
```
I recommend you turn this into a single step with 2 bullet points because a procedure with 2 bullets looks like an error.